pkgs/top-level/aliases.nix

```diff
@@ -49,11 +49,11 @@ mapAliases ({
   bashCompletion = bash-completion; # Added 2016-09-28
   bridge_utils = bridge-utils;  # added 2015-02-20
   btrfsProgs = btrfs-progs; # added 2016-01-03
-  buildbot = pythonPackages.buildbot; # added 2018-10-11
-  buildbot-full = pythonPackages.buildbot-full; # added 2018-10-11
-  buildbot-pkg = pythonPackages.buildbot-pkg; # added 2018-10-11
-  buildbot-ui = pythonPackages.buildbot-ui; # added 2018-10-11
-  buildbot-worker = pythonPackages.buildbot-worker; # added 2018-10-11
+  buildbot = python3Packages.buildbot; # added 2018-10-11
+  buildbot-full = python3Packages.buildbot-full; # added 2018-10-11
+  buildbot-pkg = python3Packages.buildbot-pkg; # added 2018-10-11
+  buildbot-ui = python3Packages.buildbot-ui; # added 2018-10-11
+  buildbot-worker = python3Packages.buildbot-worker; # added 2018-10-11
```

Comment:
Since this breaks backward compatibility for python modules, this is a good chance to drop these aliases. The aliases that provide executables should be switched to use `toPythonApplication` [1] and go to all-packages.nix others should be dropped.
[1] https://nixos.org/nixpkgs/manual/#topythonapplication-function

Reply:
Done